### ISSUE TITLE
Valid US Phone Number reporting as invalid

### ISSUE THREAD

**Issue by tesshead-dox** [daddyz/phonelib#289]
I'm using phonelib v0.8.5 and trying to validate these three numbers:
```
2276310344
3696441631
3697958630
``` 

If I run them through the google [libphonenumber](https://libphonenumber.appspot.com/) that this gem is based on they appear as valid, however this phonelib gem states they're invalid. Area code 227 is Maryland and 369 is California. Adding a 1 to the number value did not fix the issue either. 

example:
```
dev [10] pry(main)> Phonelib.parse(3696441631).valid_for_country?("US")
=> false
dev [11] pry(main)> Phonelib.parse(2276310344).valid_for_country?("US")
=> false
dev [12] pry(main)> Phonelib.parse(3697958630).valid_for_country?("US")
=> false
```

**Comment by bergey** [comment]
Is the script to import data from libphonenumber included in this repo?  I also encountered issues with 227 numbers, and confirmed that it was [recently added to libphonenumber](https://github.com/google/libphonenumber/commit/75f99bd85a47462629a9423f0c4e1bca3f3dcb64#diff-909ef29d3bec6996793f88788faa5dc5b357950b1387321197fe4fb79fdb10f6R2323).  I'd like to update the data in my application, and open a PR here if you would like, but I don't see instructions in the readme.  cc @jtangqt

**Comment by bergey** [comment]
Thanks!  That worked: https://github.com/daddyz/phonelib/pull/291

**Comment by cdcooksey** [comment]
Hello! 👋  Should this issue be closed since #291 has been merged?

**Comment by tesshead-dox** [comment]
These phone numbers work now, thank you all very much! Closing this issue.
